.github/workflows/heal-release.yaml

```diff
@@ -0,0 +1,16 @@
+name: Release Helm Charts
+on:
+  workflow_dispatch:
+
+jobs:
+  call-workflow-passing-data:
+    name: Release & Publish Helm chart
+    uses: bakdata/ci-templates/.github/workflows/helm-multi-release.yaml@main
```

Comment:
Please use a tag version instead of main

Reply:
https://github.com/bakdata/ci-templates should release a new tag here?